railgun/internal/cmd/rootcmd/run.go

```diff
@@ -11,8 +11,9 @@ func Run(ctx context.Context, c *manager.Config) error {
 	if err := StartAdmissionServer(ctx, c); err != nil {
 		return fmt.Errorf("StartAdmissionServer: %w", err)
 	}
-	if err := StartProfilingServer(ctx, c); err != nil {
-		return fmt.Errorf("StartProfilingServer: %w", err)
+	diag, err := StartDiagnosticsServer(ctx, manager.DiagnosticsPort, c)
+	if err != nil {
+		return fmt.Errorf("StartDiagnosticsServer: %w", err)
```

Comment:
This might be a language improvement to better indicate that a failure occurred with the diagnostics server in the event that the error produced is very terse:

```suggestion
		return fmt.Errorf("failed to start diagnostics server: %w", err)
```